packages/demo/src/components/component-cards/connect-card.tsx

```diff
@@ -29,17 +49,52 @@ const PROPS: PropDefinition[] = [
   },
   {
     name: "balanceType",
-    type: '"cadence" | "evm" | "vault"',
+    type: '"cadence" | "evm" | "combined"',
     required: false,
-    description: "Type of balance to display (from cross-VM token balance)",
+    description:
+      "Type of balance to display: cadence (Cadence VM only), evm (EVM only), or combined (sum of both)",
     defaultValue: '"cadence"',
   },
+  {
+    name: "balanceTokens",
+    type: "TokenConfig[]",
+    required: false,
+    description:
+      "Array of tokens with dropdown selector (first token is default). Each token needs symbol, name, and vaultIdentifier or erc20Address",
+  },
 ]
 
 export function ConnectCard() {
   const {darkMode} = useDarkMode()
   const {data: chainId, isLoading} = useFlowChainId()
   const isEmulator = chainId === "emulator" || chainId === "local"
+  const [showMultiToken, setShowMultiToken] = useState(false)
+  const [balanceType, setBalanceType] = useState<
+    "cadence" | "evm" | "combined"
+  >("cadence")
+
+  const multiTokens = [
+    {
+      symbol: "FLOW",
+      name: "Flow Token",
+      vaultIdentifier: `A.${chainId === "testnet" ? "7e60df042a9c0868" : "1654653399040a61"}.FlowToken.Vault`,
+    },
+    {
+      symbol: "USDC",
+      name: "USD Coin",
+      vaultIdentifier: `A.${chainId === "testnet" ? "b7ace0a920d2c37d" : "1e4aa0b87d10b141"}.USDCFlow.Vault`,
+      erc20Address:
+        chainId === "testnet"
+          ? undefined
+          : "0xF1815bd50389c46847f0Bda824eC8da914045D14",
+    },
+    {
+      symbol: "USDT",
+      name: "Tether USD",
+      vaultIdentifier: "A.1e4aa0b87d10b141.USDTFlow.Vault",
+      erc20Address: "0x674843C06FF83502ddb4D37c2E09C01cdA38cbc8",
```

Comment:
what happens on other networks?

Reply:
They need to put a tokenConfig that is appropriated for the selected network, so in case it's testnet the tokenconfig should have testnet addresses and same for mainnet. consider that usually this data once is set never changes because except for rare cases like the playground or dashboards/explorers you will never change network in a dapp, but even in that case it would still work, i've tried in the playground